lib/ethpy/ethpy/base/transactions.py

```diff
@@ -161,14 +171,13 @@ def retry_preview_check(exc: Exception) -> bool:
             retry_preview_check,
             function.call,
             {"from": signer_address},
-            **fn_kwargs,
+            block_identifier=block_identifier,
```

Comment:
can you make the transaction dict more explicit and add a comment above line 169? This will help us down the road when we realize we need to add more kwargs to the `call` method, like max gas or whatever.

```python
# the transaction parameter is a kwarg dictionary used for Eth.send_transaction
transaction_kwargs = {"from": signer_address}
 return_values = retry_call(
            READ_RETRY_COUNT,
            retry_preview_check,
            function.call,
            transaction_kwargs,
            block_identifier=block_identifier,
. . .
```